src/LegendDSP.jl

```diff
@@ -47,5 +47,9 @@ include("haar_filter.jl")
 include("ml.jl")
 include("dsp_ml_routines.jl")
 include("multi_intersect.jl")
+include("moving_window_multi.jl")
+include("ms_filter.jl")
```

Comment:
Can we put the filters `ms`, `sgw` and `wh` filter into one file? That would simplify the package strucutre to me.